packages/vitest/src/node/pools/forks.ts

```diff
@@ -9,6 +9,7 @@ import type { PoolProcessOptions, ProcessPool, RunWithFiles } from '../pool'
 import type { WorkspaceProject } from '../workspace'
 import { envsOrder, groupFilesByEnv } from '../../utils/test-helpers'
 import { groupBy, resolve } from '../../utils'
+import { wrapForksConfig } from '../../runtime/workers/utils'
```

Comment:
It might be bad to import from `runtime/workers` on the main thread's side. Any better place for this?

Reply:
`utils/config-helpers.ts` looks better.